Libraries/src/net/relinc/libraries/sample/TorsionSample.java

```diff
@@ -52,7 +56,44 @@ public String getParametersForPopover(boolean selected2) {
 		return "";
 	}
 	
-	public double getMeanRadius()
+	public double getStrainRate(double reflectedStrain) {
+		return 4 * (this.getAverageDiameter() / 2) * this.getWavespeed() / (this.barSetup.IncidentBar.diameter / 2 * this.getLength()) * reflectedStrain;
+	}
+	
+	public double getLoad(double strain) {
+		return this.getTorque((1) * strain) / (this.getAverageDiameter() / 2.0);
```

Comment:
no times strain here.

Reply:
This works if the input is bar strain, this should be clearly stated.

Reply:
#Important function
function polar_moment_inertia_bar(bar_radius)
    π*bar_radius^4/2
end
#Important function
function surface_strain_to_bar_torque(surface_strain,bar_radius,shear_modulus)
    torque_bar=(shear_modulus|>u"N/m^2")*polar_moment_inertia_bar(bar_radius)*surface_strain/bar_radius
end